webpack.config.js

```diff
@@ -44,6 +44,12 @@ module.exports = {
         test: /\.js$/i,
         exclude: /node_modules\//,
         use: `babel-loader`
+      },
+      {
+        test: /\.css$/,
```

Comment:
The css processing bit can be removed.

Reply:
I think I need to update react-ebi-species to the latest first? I'm still using the old one which needs CSS.